### PR TITLE
Upgrade ingress nginx version to 0.25

### PR DESCRIPTION
**Pull request by orangedeng**
Related issue:
https://github.com/rancher/rancher/issues/21826

### CODE REVIEW

**Comment by loganhz** [comment]
https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.25.1 fixed 5 CVE issues. So we need to use 0.25.1

**Comment by cjellick** [comment]
@cloudnautique can you or @drpebcak review?

File: Dockerfile.dapper [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:16.04
```
Comment (cloudnautique) [inline Dockerfile.dapper]: Should we use 18.03?

Comment (orangedeng) [reply Dockerfile.dapper]: Updated

**Pull request merged**